src/api/endpoints/healthcheck_email/healthcheck_email.py

```diff
@@ -0,0 +1,55 @@
+"""Module to test Notifications being sent out."""
```

Comment:
If this module is for testing, doesn't it belong to the testing package then?

Reply:
Its an endpoint that can be hit to confirm if the notifications are running, Just like a health-check.